bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -40,15 +40,15 @@
       <header role="banner">
         <p class="notification">
           {{ _('Heads up! Firefox Hello will be removed from Firefox soon.') }}
-          <a rel="external" class="more" href="https://support.mozilla.org/kb/hello-status">{{ _('Learn more') }}</a>
+          <a rel="external" class="more" href="https://support.mozilla.org/kb/hello-status" data-cta-position="Nav">{{ _('Learn more') }}</a>
```

Comment:
This page is being removed in #4315, so I don't think it's worth updating here in this pull request.